daemon/src/lib.rs

```diff
@@ -298,6 +303,10 @@ where
 
         tasks.add(oracle_ctx.run(oracle_constructor(cfds, Box::new(fan_out_actor))));
 
+        cfd_actor_addr
+            .do_send_async(taker_cfd::AutoRollover)
```

Comment:
Why is this usage not flagged by clippy :cry: 

Are you rebased onto latest master?

Reply:
Needs to use `send`.

Reply:
When I wrote this code we still had `do_send_async` `Sync` messages in `lib.rs` - I thought this was a concious decision to not block the startup. 
We did rebase on `master` yesterday, but the removal of `do_send_async` + clippy rule landed later. I'll fix it :)